libraries/botbuilder-core/tests/telemetryMiddleware.test.js

```diff
@@ -3,97 +3,164 @@
 
 const assert = require('assert');
 const { TestAdapter, TelemetryLoggerMiddleware, ActivityTypes } = require('../');
-const { TeamsChannelData } = require('../../botframework-schema/');
 
-describe(`TelemetryMiddleware`, function () {
+class OverrideReceiveLogger extends TelemetryLoggerMiddleware {
```

Comment:
green section is all no-op. Moved from bottom of file so get rid of "xyz is called before it's defined error"